src/ExpressionEvaluator/Core/Source/ExpressionCompiler/EEMetadataReferenceResolver.cs

```diff
@@ -82,7 +70,17 @@ public override int GetHashCode()
                         throw ExceptionUtilities.UnexpectedValue(compareResult);
                 }
             }
+
             return best;
         }
+
+        public override ImmutableArray<PortableExecutableReference> ResolveReference(string reference, string? baseFilePath, MetadataReferenceProperties properties)
```

Comment:
>public [](start = 8, length = 6)

Consider moving these back to the original location above so the difference is clear.